velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -431,7 +431,8 @@ class PlanBuilder {
       const std::vector<std::string>& sortBy,
       const dwio::common::FileFormat fileFormat =
           dwio::common::FileFormat::DWRF,
-      const std::vector<std::string>& aggregates = {});
+      const std::vector<std::string>& aggregates = {},
+      const std::string& connectorId = "test-hive");
```

Comment:
should this default to `kHiveConnectorId` instead?

Reply:
Wanted to avoid including a header in the PlanBuilder header. I plan to add a tableWriter builder soon. That should remove this.